e2e_test/batch/basic/array.slt.part

```diff
@@ -1,3 +1,5 @@
+control sortmode rowsort
```

Comment:
Remove this? This is not querying from mv.

Reply:
They are same. But as we have discussed, we should not use `rowsort` as default behavior, which is error prone.